### PR TITLE
prevent tokens from being sent to the dao itself

### PR DESCRIPTION
**Pull request by p-s-dev**

### CODE REVIEW

**Comment by CJentzsch** [comment]
why?

**Comment by D-Nice** [comment]
It dilutes the overall value of tokens, and they won't contribute to votes.

**Comment by p-s-dev** [comment]
1. People send on accident, thinking that is how to vote
2. mist doesn't sum tokens correctly when dao holds them (granted, that is a mist "bug")

**Comment by p-s-dev** [comment]
Currently, people have accidentally sent 56,828 dao tokens (worth ~$7,330) to TheDao.

File: DAO.sol [hunk 1]
```diff
@@ -740,6 +740,7 @@ contract DAO is DAOInterface, Token, TokenCreation {
         if (isFueled
             && now > closingTime
             && !isBlocked(msg.sender)
+            && _to != address(this)
```
Comment (CJentzsch) [inline DAO.sol]: you would allso need that change in `transferFrom`

Comment (p-s-dev) [inline DAO.sol]: Done, added to transferFrom() also

**Pull request merged**